README.md

```diff
@@ -1 +1,8 @@
 # gluster-mixins
+## Prerequisites
+1. Jsonnet
+
+## Install
```

Comment:
Install of what?

Reply:
changed to "How to use", similar to k8s-mixin.